OneSky.Services.Tests/Airspace/AirspaceDocExamples.cs

```diff
@@ -37,5 +37,164 @@ public void TestFindAllAirspacesWithin1Km()
             var expected = JsonConvert.DeserializeObject<AirspaceIdResult>(TestHelper.AirspaceSelectAirspacesDocExample);
             results.Should().BeEquivalentTo(expected);
         }
+
+        [Test]
+        public void TestAirspaceIntersectionFromRoute()
```

Comment:
This all looks good. As a future consideration, test methods could be named differently to better describe which aspect of a thing they are covering.

```
[Test] public void RouteShouldIntersectAirspace()
[Test] public void RouteShouldNotIntersectAirspaceWhenFooHappens()
[Test] public void FooMethodThrowsWhenBarCondition()
```